### ISSUE TITLE
When I trained the code on my dataset, the mAP always kept 0.0.

### ISSUE THREAD

**Issue by Williamwsk** [toandaominh1997/EfficientDet.Pytorch#140]
When I trained the code on my dataset, the mAP always kept 0.0. My dataset had 17 classes, how can I fix it? Hope to your reply.
`mAP:
len(VOC_CLASSES): 17
A: 0.0
B: 0.0
C: 0.0
D: 0.0
T: 0.0
F: 0.0
score: 0.0
anum: 0.0
bnum: 0.0
cnum: 0.0
rnum: 0.0
onum: 0.0
title: 0.0
keyword1: 0.0
keyword2: 0.0
keyword3: 0.0
keyword4: 0.0
avg mAP: 0.0
55 epoch:        start training....`

**Comment by Wangyy-moya** [comment]
I got the same issue.hope the author reply

**Comment by ZixuanLingit666** [comment]
I also have this problem, but my  loss in trainning is normal

**Comment by ProjectCase** [comment]
I have the same problem, the mAP is always 0, but the loss is converging normally
